node_modules/oae-core/about/css/about.css

```diff
@@ -0,0 +1,62 @@
+
+#about-description .about-description-empty,
+#about-description .about-description-empty a {
+    color: #888;
+}
+
+#about-description .about-description-empty a {
+    font-style: italic;
+    text-decoration: underline;
+}
+
+#about-members {
+    margin-top: 35px;
+}
+
+#about-members > div.about-show-all {
+    margin-top: 15px;
+}
+
+#about-members .about-members {
+    display: -webkit-box;
```

Comment:
I'm pretty sure we don't need any of these `display` and `flex-wrap` properties. Something like a `display: inline-block` should be sufficient